.github/ISSUE_TEMPLATE/bug-or-issue.yaml

```diff
@@ -0,0 +1,70 @@
+# Copyright The Notary Project Authors.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: 🐛 Bug or Issue
+description: Something is not working as expected or not working at all! Report it here!
+labels: [bug, triage]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for taking the time to fill out this issue report. 🛑 Please check existing issues first before continuing: https://github.com/notaryproject/notation/issues
+  - type: dropdown
+    id: area
+    validations:
+      required: true
+    attributes:
+      label: "What is the areas you experience the issue in?"
+      options: 
+        - Notation CLI
+        - Notation-Core-Go Library
+        - Notation-Go Library
```

Comment:
Since there is an issue template per repository, we don't need this question any more.